src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -36,12 +55,12 @@ const sendFile = require('send')
 const LRU = require('lru-cache')
 const crypto = require('crypto')
 
-const calculateQueryHash = queryString => crypto.createHash('sha1').update(queryString).digest('base64')
+const calculateQueryHash = (query: string) => crypto.createHash('sha1').update(query).digest('base64')
```

Comment:
I prefer `queryString: string` since "query string" is what I would call it aloud. "query" has a different meaning to me.